dist/css/application.css

```
@@ -1,37 +1,168 @@
-.govuk-width-container {
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  padding: 0;
+  margin: 0;
+}
+
+p {
+  padding: 0;
+  margin: 0;
+}
+
+p + p {
+  margin-bottom: .5em;
+}
+
+.site-header {
+  background-color: #ffcd4e;
+  padding: 1em 0;
+  border-bottom: 5px solid #f78932;
+}
+
+.width-container {
   max-width: 95%;
+  margin: 0 auto;
+  padding: 0;
+}
+
+h1 {
+  font-size: 2rem;
+}
+
+h2 {
+  font-size: 1.5rem;
 }
 
-.text-center {
+h3 {
+  font-size: 1.2rem;
+}
+
+.heading {
+  margin: 0;
+  padding: 0;
```

```suggestion
  box-shadow: 0px 10px 10px -5px #ddd;
  margin: 0;
  margin-bottom: .5em;
  padding: .5em;
```

What do you think?

Little eye candy, making some space between the edges of the div (specially the top one), as well as making it simpler to see the heading when sticky?

---

sounds good, will check out the change tmrw